syntax = "proto3";

package example;

service HelloWorldService {
    rpc SayHello (HelloRequest) returns (HelloResponse) {}
}

message HelloRequest {
    string message = 1;
}

message HelloResponse {
    string message = 1;
}
